SpatialGDK/Extras/internal-documentation/release-process.md

```diff
@@ -152,18 +131,24 @@ call "%~dp0ProjectPaths.bat"
 
 ## Release
 
-All of the above tests must have passed and there must be no outstanding blocking issues before you start this, the release phase.
+All of the above tests **must** have passed and there must be no outstanding blocking issues before you start this, the release phase.
```

Comment:
Line  116: ## Validation (GDK Starter Template)
This talks about the starter template but is actually referring to UnrealGDKThirdPersonShooter which is incorrect. StarterTemplate is a different flow that uses the template built into our fork of the UnrealEngine

Reply:
Line 117:  * Your final script shoudl look somethisn like: `@echo off

Should be:
** * Your final script should look something like: `@echo off**

Reply:
fixed